front-end-admin/src/components/layout/Navigation.vue

```diff
@@ -14,6 +14,12 @@
       <div id="navbar" class="navbar-collapse collapse">
         <slot />
       </div>
+      <div>
```

Comment:
I like the look as is, but if you were interested in looking at some bootstrap buttons check these out

https://getbootstrap.com/docs/4.0/components/buttons/